src/app/sidenav/sidenav.component.ts

```diff
@@ -9,40 +13,60 @@ import {Utils} from '../utils';
 })
 export class SidenavComponent implements OnInit {
   @ViewChild('sidenav') sidenav: MatSidenav;
-  communities: Array<any> = [];
-  communitiesFavourited: Array<any> = [];
+  communities: Array<Community> = [];
+  communitiesFavourited: Array<Community> = [];
 
   get sidebarWidth(): number {
     return 300;
   }
 
   utils = Utils;
 
-  constructor() { }
+  constructor(private _UserService: UserService, private _CommunityService: CommunityService) { }
```

Comment:
nit: The general naming scheme for DI'd services is `userService` or `communityService` camel case.